tasks/kernel_matrix_testing/init_kmt.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
+from .tool import Exit
+from .download import download_rootfs, download_kernel_packages
+import os
+import getpass
+
+KMT_DIR = os.path.join("/", "home", "kernel-version-testing")
+KMT_ROOTFS_DIR = os.path.join(KMT_DIR, "rootfs")
+KMT_STACKS_DIR = os.path.join(KMT_DIR, "stacks")
+KMT_PACKAGES_DIR = os.path.join(KMT_DIR, "kernel-packages")
+KMT_BACKUP_DIR = os.path.join(KMT_DIR, "backups")
+KMT_LIBVIRT_DIR = os.path.join(KMT_DIR, "libvirt")
+KMT_SHARED_DIR = os.path.join("/", "opt", "kernel-version-testing")
+KMT_KHEADERS_DIR = os.path.join("/", "opt", "kernel-version-testing", "kernel-headers")
+
+VMCONFIG = "vm-config.json"
+
+priv_key = """-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
Is it expected to have a private key here ?

Reply:
we usually share them through 1Password

Reply:
This is the private key for the micro-vms we launch. It is fine to make it public. We want to automatically allow access to anyone setting up the VMs locally.